lib/tests/formulae.rb

```diff
@@ -225,19 +225,66 @@ def livecheck(formula)
         return if livecheck_step.failed?
         return unless livecheck_step.output?
 
-        livecheck_info = JSON.parse(livecheck_step.output)
-        return unless livecheck_info.first["version"]["newer_than_upstream"]
+        livecheck_info = JSON.parse(livecheck_step.output)&.first
+        if livecheck_info.blank?
+          # This warning shouldn't appear under normal circumstances. If we
+          # encounter this, something unexpectedly changed/broke in livecheck.
+          no_info_msg = "The data from livecheck is unexpectedly empty"
+          if ENV["GITHUB_ACTIONS"].present?
+            puts GitHub::Actions::Annotation.new(
+              :warning,
+              no_info_msg,
+              title: "#{formula}: livecheck data is empty",
+              file:  formula.path.to_s.delete_prefix("#{repository}/"),
+            )
+          else
+            opoo no_info_msg
+          end
+        end
```

Comment:
Not totally convinced this is necessary to handle and downgrade to a green CI warning?

Reply:
It's a situation that shouldn't ever occur (i.e., livecheck uses `rescue` and would surface an error in the JSON output) and it would indicate that something needs to be fixed, so I'm good with letting an error occur if `livecheck_info` is ever blank. I'll remove this error handling.